atest/testdata/test_libraries/LibUsingLoggingApi.py

```diff
@@ -24,9 +24,13 @@ def write_messages_to_console():
     logger.console('in two parts')
     logger.info('To log and console', also_console=True)
 
+
 def log_non_strings():
     logger.info(42)
     logger.warn(True)
 
+
 def log_callable():
     logger.info(log_callable)
+    
+
```

Comment:
Empty rows at the end should be avoided. They add noise to diffs. The last line should end with a newline, though. (Again something that IDEs can do automatically and something that should be mentioned in contribution guidelines.)